MediaBrowser.Model/Session/PlaystateCommand.cs

```diff
@@ -48,6 +48,21 @@ public enum PlaystateCommand
         /// <summary>
         /// The play pause.
         /// </summary>
-        PlayPause
+        PlayPause,
+
+        /// <summary>
+        /// The mute.
+        /// </summary>
+        Mute,
+
+        /// <summary>
+        /// The unmute.
+        /// </summary>
+        Unmute,
+
+        /// <summary>
+        /// The toggle mute.
+        /// </summary>
+        ToggleMute
```

Comment:
Changing this enum affects one of the API endpoints which I think is not intended. To mute/unmute/togglemute with the API you can already use the `/Sessions/[sessionid]/Command` endpoint.

Reply:
Oh interesting. So unlike with fast forwarding, rewinding, playing, pausing, and seeking, muting/un-muting or toggling muting does not need to be included in the `PlaystateCommand` enum. I can definitely revert that change.

Reply:
Hmm, removing those values from the enum made a test fail that didn't fail with a previous commit. Are we sure that we don't need them?